usagov_benefit_finder/modules/usagov_benefit_finder_api/src/Controller/LifeEventController.php

```diff
@@ -374,7 +374,7 @@ public function buildCriteriaFieldset($criteria) {
     $criteria_fieldset = [
       "criteriaKey" => current($criteria->get('field_b_criteria_key')->referencedEntities())->get('field_b_id')->value,
       "legend" => $criteria->get('field_b_legend')->value ?? "",
-      "required" => $criteria->get('field_b_required')->value ? "TRUE" : "FALSE",
+      "required" => $criteria->get('field_b_required')->value ? "true" : "false",
```

Comment:
> A value can be a string in double quotes, or a number, or true or false or null, or an object or an array

The change request is an attempt to remove the string value, and instead return boolean `true` or `false`

Is there any reason we can't do this?

Reply:
Changed from string to boolean.